app/src/main/res/values/strings.xml

```diff
@@ -956,4 +956,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_shop_gone_replaced_answer_hint">Type kind of place here</string>
     <string name="quest_shop_gone_replaced_answer_error">"Don't know what this is. Consider leaving a note."</string>
     <string name="quest_shop_vacant_type_title">This shop has been vacant. What\'s here now?</string>
+    <string name="quest_construction_completes_at_known_date">Completes at known date</string>
+    <string name="quest_construction_completion_date_title">When object will be open?</string>
```

Comment:
```suggestion
    <string name="quest_construction_completion_date_title">When will it be complete?</string>
```

Though, not sure if a description here is necessary at all. After all, it should come to no surprise to the user to be asked to provide a date when he previously selected "Completes at known date"

Reply:
Sometimes I missclick or get interrupted and forget what I was doing, so I quite like to have confirmation what I am doing on every screen. But I am not strongly attached to it.